src/components/CertificateDropZone/CertificateDropZone.tsx

```diff
@@ -14,12 +14,13 @@ interface CertificateDropzoneProps {
 export const CertificateDropZone: FunctionComponent<CertificateDropzoneProps> = (props) => {
   const { toggleQrReaderVisible } = props;
   const dispatch = useDispatch();
-  const { verificationPending, verificationStatus, retrieveCertificateByActionState } = useSelector(
+  const { verificationPending, retrieveCertificateByActionState, verificationStatus, verificationError } = useSelector(
     (state: RootState) => state.certificate
   );
 
   const isVerificationPending = verificationPending;
-  const isVerificationError = verificationStatus && !isValid(verificationStatus);
+  const isVerificationError =
+    (verificationError && verificationError.length > 0) || (verificationStatus && !isValid(verificationStatus));
```

Comment:
you want to change this into a small function instead? and see the comment below also

Reply:
ok